packages/starknet-snap/src/getErc20TokenBalance.ts

```diff
@@ -50,13 +52,19 @@ export async function getErc20TokenBalance(params: ApiParams) {
       `getErc20Balance:\nerc20Address: ${erc20Address}\nuserAddress: ${userAddress}`,
     );
 
-    const resp = await callContract(network, erc20Address, 'balanceOf', [
-      numUtils.toBigInt(userAddress).toString(10),
-    ]);
+    const blockIdentifier = (await isAccountDeployed(network, userAddress))
+      ? 'pending'
```

Comment:
can we have the pending and latest in constant / enum ? 

e.g 
enum BlockIdentifier {
  Latest: 'latest',
  Pending: 'pending'
}

Reply:
btw, why we need isAccountDeployed in here?

Reply:
add back unit test